.changeset/nervous-clocks-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui': patch
+---
+
+Fixed issue when customer is using auto confirmed lambda. Now it will work again.
```

Comment:
Can you please add more descriptive detail on what the error was and how it was fixed for the changelog?

Reply:
OK, I'll add it.

Reply:
Ok updated